CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # CHANGELOG
 
+## v1.5.2
+
+* Add access token refresh error class that includes HTTP status (#310)
+* Python3 compatibility fixes (#316, #318, #329)
```

Comment:
329 can be left out. #316 and #318 should be highlighted as Python3 compat for the django stuff